modules/sor/utils/data.ts

```diff
@@ -58,7 +59,7 @@ async function getPoolsByIds(
     poolIds: string[],
 ): Promise<PrismaPoolAndHookWithDynamic[]> {
     const pools = await prisma.prismaPool.findMany({
-        where: { id: { in: poolIds }, chain, protocolVersion, type },
+        where: { id: { in: poolIds }, chain, protocolVersion, type: { in: [...type.in, 'LIQUIDITY_BOOTSTRAPPING'] } },
```

Comment:
Does 'LIQUIDITY_BOOTSTRAPPING' need to be manually added here if its already in type?